File: Assignment14/Question1.c
//Nested Iteration
/*
input 4 4
output 1   2    3   4
       1    2   3   4
       1    2   3   4
       1    2   3   4
       
*/
# include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0;
    int iNum = 1;
    for(i = 0 ; i < iRow ; i++)
    {    int j = 0;
        for(j = 0 ; j < iCol ; j++)
        {
            printf("%d\t",iNum++); 
        }
    printf("\n");
    }
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter the number of rows : ");
    scanf("%d",&iValue1);

    printf("Enter the number of Coloumns : ");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    
    return 0;    
}

File: Assignment50/Question1.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        int i = 0, j = 0;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                if(i + j == iCol + 1)
                {
                    System.out.print("#\t");
                }
                else
                {
                    System.out.print("*\t");
                }
            }
            System.out.println();
        }
    }
}

class Question1
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows: ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns: ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1, iValue2);
    }
}
